resources/definitions/robo_3d_r1.def.json

```diff
@@ -0,0 +1,61 @@
+        "platform_offset": [ 0, 0, 0]
+    },
+
+    "overrides": {
+        "machine_start_gcode": {
+            "default_value": " G92 E0 ;\n M565 Z-1 ;\n G1 Z5 F5000 ;\n G29 ;\n"
+        },
+        "machine_end_gcode": {
+            "default_value": " M104 S0                     ;extruder heater off\n M140 S0                     ;heated bed heater off (if you have it)\n G91                                    ;relative positioning\n G1 E-1 F300                            ;retract the filament a bit before lifting the nozzle, to release some of the pressure\n G1 Z+0.5 E-5 X-20 Y-20 F{speed_travel} ;move Z up a bit and retract filament even more\n G28 X0 Y0                              ;move X/Y to min endstops, so the head is out of the way\n M84                         ;steppers off\n G90                         ;absolute positioning\n"
+        },
+        "cool_min_layer_time": { "default_value": 7 },
+        "speed_topbottom": { "default_value": 40 },
+        "retraction_speed": { "default_value": 50 },
+        "layer_0_z_overlap": { "default_value": 0.2 },
+        "cool_min_speed": { "default_value": 19 },
+        "material_bed_temperature": { "default_value": 60 },
+        "support_angle": { "default_value": 50 },
+        "speed_layer_0": { "default_value": 30 },
+        "line_width": { "default_value": 0.4 },
+        "speed_infill": { "default_value": 60 },
+        "prime_tower_size": { "default_value": 8.660254037844387 },
+        "support_enable": { "default_value": true },
+        "cool_fan_full_at_height": { "default_value": 0.1 },
+        "material_diameter": { "default_value": 1.75 },
+        "bottom_thickness": { "default_value": 1.2 },
+        "raft_airgap": { "default_value": 0.2 },
+        "layer_height_0": { "default_value": 0.15 },
+        "top_thickness": { "default_value": 1.2 },
+        "speed_wall_0": { "default_value": 40 },
+        "retraction_min_travel": { "default_value": 5 },
+        "material_flow": { "default_value": 100 },
+        "infill_sparse_density": { "default_value": 10 },
+        "wall_thickness": { "default_value": 1.2 },
+        "material_print_temperature": { "default_value": 190 },
+        "retraction_amount": { "default_value": 3 },
+        "layer_height": { "default_value": 0.2 },
+        "speed_print": { "default_value": 40 },
+        "machine_extruder_count": { "default_value": 1 },
+        "retraction_combing": { "default_value": "off" },
+        "machine_heated_bed": { "default_value": true },
+        "machine_center_is_zero": { "default_value": false },
+        "machine_height": { "default_value": 210 },
+        "adhesion_type": { "default_value": "raft" },
+        "machine_gcode_flavor": { "default_value": "RepRap (Marlin/Sprinter)" },
+        "machine_depth": { "default_value": 245 },
+        "machine_width": { "default_value": 225 },
+        "support_z_distance": { "default_value": 0.22 },
+        "machine_name": { "default_value": "ROBO 3D R1" }
```

Comment:
Any reason why this is different from the actual name as shown in the interface?